blaze/interactive.py

```diff
@@ -229,26 +228,27 @@ def short_dshape(ds, nlines=5):
     return s
 
 
-def coerce_to(typ, x):
+def coerce_to(typ, x, odo_kwargs=None):
     try:
         return typ(x)
     except TypeError:
-        return odo(x, typ)
+        return odo(x, typ, **odo_kwargs or {})
```

Comment:
I presume the precedence here is equivalent to `**(odo_kwargs or {})`, correct?  What about putting explicit parens around the disjunction to make it more clear -- the spacing makes it look strange at first read.